settings.gradle

```diff
@@ -12,6 +15,5 @@ project(':payment-contracts').projectDir = "$rootDir/payment-contracts" as File
 include ':registration-contracts'
 project(':registration-contracts').projectDir = "$rootDir/registration-contracts" as File
 
-include ':common'
-project(':common').projectDir = "$rootDir/common" as File
-
+include ':conference-contracts'
```

Comment:
Have contracts as parts of corresponding modules coming under `contract` package in both proto and java parts of the code. It's OK to do this in the PR #7, if you'd like.